src/contracts/MigrationHelperAmm.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {IERC20WithPermit} from 'solidity-utils/contracts/oz-common/interfaces/IERC20WithPermit.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+
+import {AaveV2EthereumAMM, AaveV2EthereumAMMAssets} from 'aave-address-book/AaveV2EthereumAMM.sol';
+import {AaveV3Ethereum} from 'aave-address-book/AaveV3Ethereum.sol';
+import {DataTypes, ILendingPool as IV2Pool} from 'aave-address-book/AaveV2.sol';
+
+import {IWstETH} from '../interfaces/IWstETH.sol';
+import {MigrationHelper} from './MigrationHelper.sol';
+
+/**
+ * @title MigrationHelperMainnet
+ * @author BGD Labs
+ * @dev Contract to migrate positions from Aave v2 to Aave v3 Ethereum Mainnet pools
+ *   wraps stETH to wStETH to make it compatible
+ */
+contract MigrationHelperAmm is MigrationHelper {
+  using SafeERC20 for IERC20WithPermit;
+  using SafeERC20 for IWstETH;
+
+  constructor() MigrationHelper(AaveV3Ethereum.POOL, AaveV2EthereumAMM.POOL) {
+  }
+
+  function cacheATokens() public virtual override {
```

Comment:
overwriting as most assets on this pool are not migratable so it's a waste of gas.